test/data/resource/genesyscloud_script/test_script.json

```diff
@@ -209,7 +209,8 @@
                                 "value": {}
                             },
                             "value": {
-                                "typeName": "variable"
+                                "typeName": "variable",
+                                "value": "MyCheckBox"
```

Comment:
Had to fix the script.  Once I started publishing the script failed with a validation error because the checkbox in the script was missing a value.